runtime-light/stdlib/diagnostics/backtrace.cpp

```diff
@@ -44,8 +44,8 @@ size_t async_frames(std::span<void*> addresses, kphp::coro::async_stack_frame* t
 namespace kphp::diagnostic::impl {
 
 size_t async_backtrace(std::span<void*> addresses) noexcept {
-  if (const auto* instance_state{k2::instance_state()}; instance_state != nullptr) [[likely]] {
-    const auto& async_stack_root{instance_state->coroutine_instance_state.coroutine_stack_root};
+  if (k2::is_instance_state_available()) [[likely]] {
```

Comment:
Instead of `k2::is_instance_state_available` I'd suggest adding following function:
`std::optional<std::reference_wrapper<InstanceState>> InstanceState::try_get()`